service/history/api/create_workflow_util.go

```diff
@@ -95,6 +95,7 @@ func NewWorkflowWithSignal(
 			signalWithStartRequest.GetSignalInput(),
 			signalWithStartRequest.GetIdentity(),
 			signalWithStartRequest.GetHeader(),
+			signalWithStartRequest.GetSkipGenerateWorkflowTask(),
```

Comment:
first workflow task should not be generated if signalWithStartRequest has skip workflow task flag set.

Reply:
synced offline. workflow task for new workflow should still be generated.